lib/logdna/client.rb

```diff
@@ -15,20 +20,21 @@ def initialize(request, uri, opts)
       @buffer = []
       @buffer_byte_size = 0
 
-      @side_messages = []
-
       @lock = Mutex.new
-      @side_message_lock = Mutex.new
-      @flush_limit = opts[:flush_size] || Resources::FLUSH_BYTE_LIMIT
-      @flush_interval = opts[:flush_interval] || Resources::FLUSH_INTERVAL
-      @flush_scheduled = false
-      @exception_flag = false
+      @flush_threshold = opts[:flush_size] || Resources::FLUSH_BYTE_LIMIT
+      @flush_limit = 2 * 1024 * 1024
+
+      flush_interval = opts[:flush_interval] || Resources::FLUSH_INTERVAL
```

Comment:
Was this meant to be an instance variable?

Reply:
No, only the `@flush_scheduler` cares about the `flush_interval`.

Reply:
Since we dropped `@flush_scheduler`, this is an instance variable again.